runtime/runtime-common/Cargo.toml

```diff
@@ -46,6 +46,25 @@ pallet-receipt-registry = { workspace = true, optional = true }
 cfg-if.workspace = true
 log.workspace = true
 
+# Orml
+orml-asset-registry.workspace = true
```

Comment:
I would prefer to feature gate it until we release it to sydney to not make sydney build heavier.